autotelic/fastify-mail
# feat: more types

# Summary

Add more type declarations for plugin usage

## Test Plan

- Install this branch as a dependency for any typescript project using `fastify-mail`, for instance [fasdentify-v2](https://github.com/autotelic/fasdentify-v2/blob/develop/services/rest/package.json#L51)
- Build app and check for any type errors
